packages/debugger/lib/evm/sagas/index.js

```diff
@@ -31,16 +31,24 @@ export function* addContext(context) {
 
 /**
  * Adds known deployed instance of binary at address
+ * by default adds it to the real, internal list of instances.
+ * however it can also add it to the external display-only list of instances.
+ * (note that I'm calling this "display-only", but really they're
+ * also used for, e.g., determining what external sources need fetching)
```

Comment:
The "internal instances" are the codex.  Instances for contract creations shouldn't be added to the codex, as the code will only exist for them *after* the creation successfully completes, and the codex is supposed to reflect the state of things at the current point in the transaction.  So, they shouldn't have their bytecode added to the codex, only to affectedInstances.

Reply:
I thought about calling them "external", but with all the other uses of that word, that seemed too confusing.  Like it sounds like it might mean "external to the project" as in `external.js`.  So I settled on "display", but I agree it's not the best term.  Haven't come up with something better yet though.

Reply:
OK, renaming to "affected instances".  Duh!